### ISSUE TITLE
Re-sharing a file leads to logout

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#1053]
I know that there is/was an issue about this, but I currently cannot find it and thus do not know the state.

### Actual behaviour
- error is displayed and login screen is shown

### Expected behaviour
- correctly reshared file
 
### Steps to reproduce
1. share a file with user a
2. share the same file with user b 
3. 

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky you probably mean #730 which has been fixed via #986 and would then be fixed for 1.5.0 :)

**Comment by mario** [comment]
This is working, so closing.
